tests/acceptance/test_checkbox.py

```diff
@@ -147,3 +147,27 @@ def setUp(self):
     def test_default_selection(self):
         self.sut.send(key.ENTER)
         self.sut.expect("{'interests': ", timeout=1)
+
+
+@unittest.skipUnless(sys.platform.startswith("lin"), "Linux only")
+class CheckLockedTest(unittest.TestCase):
+    def setUp(self):
+        self.sut = pexpect.spawn("python examples/checkbox_locked.py")
+        self.sut.expect("DevOps.*", timeout=1)
+
+    def test_default_selection(self):
+        self.sut.send(key.ENTER)
+        self.sut.expect(r"{'courses': \['Programming fundamentals'\]}", timeout=1)
+
+    def test_locked_option(self):
+        self.sut.send(key.SPACE)
+        self.sut.send(key.LEFT)
```

Comment:
Its a good idea to test for both option to select a potion.

But in this case the test would actually work even if the locking code didn work, as it deselects and than imediatly selects againt. So please seperate this into two seperate tests

Reply:
Seprated into two tests